tests/utils.go

```diff
@@ -3816,12 +3838,45 @@ func RenderHostPathJob(jobName string, dir string, hostPathType k8sv1.HostPathTy
 	return job
 }
 
+func WaitForActualJobToSucceed(virtClient *kubecli.KubevirtClient, job *batchv1.Job, timeoutSec time.Duration) {
+	fail := false
+	EventuallyWithOffset(1, func() bool {
+		job, err := (*virtClient).BatchV1().Jobs(job.Namespace).Get(job.Name, metav1.GetOptions{})
+		ExpectWithOffset(1, err).ToNot(HaveOccurred())
+		if job.Status.Failed == *job.Spec.BackoffLimit {
+			fail = true
+			return fail
+		}
+		return job.Status.Succeeded > 0
+	}, timeoutSec*time.Second, 1*time.Second).Should(Equal(true))
+	Expect(fail).To(BeFalse(), "Job was expected to succeed but failed")
+}
+
+func WaitForActualJobToFailed(virtClient *kubecli.KubevirtClient, job *batchv1.Job, timeoutSec time.Duration) {
+	success := false
+	EventuallyWithOffset(1, func() bool {
+		job, err := (*virtClient).BatchV1().Jobs(job.Namespace).Get(job.Name, metav1.GetOptions{})
+		ExpectWithOffset(1, err).ToNot(HaveOccurred())
+		if job.Status.Succeeded > 0 {
+			success = true
+			return success
+		}
+		return job.Status.Failed > 0
+	}, timeoutSec*time.Second, 1*time.Second).Should(Equal(true))
+	Expect(success).To(BeFalse(), "Job was expected to fail but successed")
+}
+
 // NewHelloWorldJob takes a DNS entry or an IP and a port which it will use create a pod
 // which tries to contact the host on the provided port. It expects to receive "Hello World!" to succeed.
 func NewHelloWorldJob(host string, port string) *k8sv1.Pod {
```

Comment:
Is this still being used? If no can we remove it and rename the `NewHelloWorldActualJob` -> `NewHelloWorldJob`. If it is still being used could it be renamed to `NewHelloWorldPod`? Same with `RenderJob` vs. `RenderActualJob`? 

If we need to keep both could their names be `RenderPod` and `RenderJob`? It took me a while of reading to understand why the word `Actual` was in the function names and what that meant so I think the above could improve readability

Reply:
Thanks. I agree it was poor naming.